File: 2562 최대값/2562.cpp
#include <iostream>
int main() {
	int numbers[9];
	for (int i = 0; i < 9; ++i) {
		std::cin >> numbers[i];
	}

	int order = 0;
	int max = numbers[0];
	for (int i = 1; i < 9; ++i) {
		if (max < numbers[i]) {
			max = numbers[i];
			order = i;
		}
	}

	std::cout << max << "\n" << order + 1 << "\n";
}

File: 2562 최대값/2562.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');

let numbers = [];
input.forEach(inputElement => {
    numbers.push(parseInt(inputElement));
});

let max = numbers[0];
let order = 0;
for (let i = 1; i < 9; ++i) {
    if (max < numbers[i]) {
        order = i;
        max = numbers[i];
    }
}

console.log(max + "\n" + (order + 1));